src/Libraries/Microsoft.Extensions.Http.Telemetry/Logging/HttpClientLoggingTagNames.cs

```diff
@@ -60,6 +60,7 @@ public static class HttpClientLoggingTagNames
     /// Gets a list of all dimension names.
     /// </summary>
     /// <returns>A read-only <see cref="IReadOnlyList{String}"/> of all dimension names.</returns>
+    // TODO: should we rename Dimensions to Tags here?
     public static IReadOnlyList<string> DimensionNames { get; } =
```

Comment:
@geeknoid should I rename the property to be `TagNames`?

Reply:
Or it's okay to use `Dimensions`?

Reply:
TagNames please

Reply:
Done